client/src/domain/Ship.ts

```diff
@@ -0,0 +1,17 @@
+export default class Ship {
+  readonly length: number;
+  readonly coordinates: { x: number; y: number }[];
+  readonly hits: number;
+  readonly isDestroyed: boolean;
+
+  constructor(
+    length: number,
+    coordinates: { x: number; y: number }[],
+    hits: number
+  ) {
+    this.length = length;
+    this.coordinates = coordinates;
+    this.hits = hits;
+    this.isDestroyed = false;
```

Comment:
nžn ar čia pamiršai isDestroyed konstruktoriaus param duoti. + reiktų čj vadint patį variable "destroyed", o jeigu būtų getteris tai tada tas "is" prisidėtų